### PR TITLE
Binary classifier notebook documentation update

### LINKED ISSUE #8: missing full config data in Binary Classifier Notebook example
## Problem Summary
If you follow the Binary Classifier Notebook (BCN) example directly you won't download the proper config files (as described here: https://epic-open-source.github.io/seismometer/user_guide/index.html#config-files). 

## Impact
Users aren't able to easily follow the tutorial.

## Possible Solution
Placing them in the https://github.com/epic-open-source/seismometer-data repo and updating the example to download those files should fix the issue.

## Steps to Reproduce
1. Follow the BCN example in a clean directory.

## Suggested fix
See above.

### PR DESCRIPTION
**Pull request by MahmoodEtedadi**
Clarify assumptions about the required files to run the example notebook, closes #8.

### CODE REVIEW

**Review by nhayes-rich — CHANGES_REQUESTED** [review]
Do you want to also add a section at the top of the example notebooks rst file explaining the breakdown of the files (i.e., which one is where and why they're separate)?

**Comment by diehlbw** [comment]
Can you create a newsfragment for this and reference the issue?  
My PR just snuck in which starts enforcing these

File: docs/example_notebooks/index.rst [hunk 1]
```diff
@@ -3,5 +3,38 @@
...
+directory as the notebook: configuration files and data.
+
+Configuration Files
+~~~~~~~~~~~~~~~~~~~
+
+The following configuraion files are required to be in the same directory as the notebook: 
+
+- ``config.yml``, 
+- ``usage_config.yml``,
+- ``data/metada.json``.
```
Comment (nhayes-rich) [inline docs/example_notebooks/index.rst:19]:
```suggestion
- ``data/metadata.json``.
```

File: docs/example_notebooks/index.rst [hunk 2]
```diff
@@ -3,5 +3,38 @@
 Example Notebooks
 =================
 
+Directory Structure
+-------------------
+
+To run the notebook, we assume the existence of the following files in the same 
+directory as the notebook: configuration files and data.
```
Comment (nhayes-rich) [inline docs/example_notebooks/index.rst:10]:
```suggestion
directory as the notebook: configuration files and the data directory.
```

File: docs/example_notebooks/index.rst [hunk 3]
```diff
@@ -3,5 +3,38 @@
...
+- ``data/metada.json``.
+
+.. seealso::
+    :ref:`config-files` for more information on configuration files.
+
+
+Data
+~~~~
+
+The first code cell in the notebook downloads the required data from
```
Comment (nhayes-rich) [inline docs/example_notebooks/index.rst:28]:
```suggestion
The first code cell in the notebook downloads the required data and data dictionary from 
```

**Review by nhayes-rich — CHANGES_REQUESTED** [review]

File: docs/example_notebooks/index.rst [hunk 4]
```diff
@@ -3,5 +3,38 @@
...
+Directory Structure
+-------------------
+
+To run the notebook, we assume the existence of the following files in the same 
+directory as the notebook: configuration files and the data directory.
+
+Configuration Files
+~~~~~~~~~~~~~~~~~~~
+
+The following configuraion files are required to be in the same directory as the notebook:
```
Comment (nhayes-rich) [inline docs/example_notebooks/index.rst:15]:
```suggestion
The following configuration files are required to be in the same directory as the notebook: 
```

**Review by nhayes-rich — APPROVED** [review]

**Review by diehlbw — APPROVED** [review]

**Pull request merged**